<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomSearchWidget</class>
 <widget class="QWidget" name="CustomSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomSearchWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="searchLabel">
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="searchLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="resultsListWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>66</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>searchLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>resultsListWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
